run_fastsurfer.sh

```diff
@@ -875,6 +886,23 @@ if [[ "$run_seg_pipeline" == "1" ]]
                 exit 1
             fi
         fi
+
+        if [[ -n "$t2" ]]
+        then
+          # ... we have a t2 image, bias field-correct it
+        echo "INFO: Running N4 bias-field correction of the t2" | tee -a "$seg_log"
+        cmd=($python "${reconsurfdir}/N4_bias_correct.py" "--in" "$t2"
```

Comment:
Have you ever tested our N4 wrapper script for the T2 image?

Reply:
@santiestrada32 has used the script for this (we even found a bug in there doing that). But this PR also still needs some testing anyways...